packages/backend/src/handlers/actions/createQuest/createQuest.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Quest_Insert_Input,
+  QuestRepetition_Enum,
+} from '../../../lib/autogen/hasura-sdk';
+import { getClient } from '../../../lib/hasuraClient';
+import { CreateQuestInput, CreateQuestOutput } from '../types';
+
+export async function createQuest(
+  playerId: string,
+  quest: CreateQuestInput,
+): Promise<CreateQuestOutput> {
+  const client = getClient();
+
+  if (quest.repetition && !(quest.repetition in QuestRepetition_Enum)) {
```

Comment:
¿If `quest.repetition` is falsey, won't `quest.repetition in QuestRepetition_Enum` be false as well, meaning this could just be `!(quest.repetition in QuestRepetition_Enum)`?

Reply:
`repetition`  is `QuestRepetition_Enum | null`, so if `repetition` is `null`, it can't be in `QuestRepetition_Enum` for typescript.

Reply:
> > Nothing looks broken to me. Just had a couple potentially shorter expressions of some logic.
> 
> This is a draft pull request marked as Work in progress, not ready to be reviewed yet. You can have a look at this instead: #339

"`[WIP]`". Gotcha. I was unfamiliar with the process.